src/vl-select-location.js

```diff
@@ -85,6 +81,10 @@ class VlSelectLocation extends VlSelect {
     this._changeTranslation('select.no_more_options', value);
   }
 
+  set searchPlaceholder(value) {
+    this.setAttribute('data-vl-search-placeholder', value);
+  }
+
```

Comment:
Analoge opmerking als bij vl-select zelf: zolang we hier geen proza keys kunnen meegeven zijn we niet waar we uiteindelijk willen zijn. Een stapje dichter maar niet waar we willen zijn denk ik. 
Dus da's altijd ook functioneel met Kenzo mee te nemen of dit goed genoeg is, dat er al onderscheid tussen de vl-select-location en de andere vl-selects, maar bij geen van beide zitten de placeholders dan in proza ...

Reply:
Ik heb net met Kenzo afgestemd dat de default teksten voorlopig voldoende zijn en dat er nog geen proza integratie nodig is.